firmware/ventilator-controller-stm32/Core/Src/main.cpp

```diff
@@ -338,7 +346,7 @@ PF::Driver::Power::Simulator power_simulator;
 
 // Initializables
 auto initializables =
-    PF::Driver::make_initializables(sfm3019_air, sfm3019_o2, fdo2, nonin_oem, ltc4015);
+    PF::Driver::make_initializables(sfm3019_air, sfm3019_o2, fdo2, nonin_oem, ltc4015, abp);
```

Comment:
Let's move `abp` to be either right before `fdo2` or right after `fdo2`, so that all sensors for the breathing circuit are next to each other in this list.